apps/public-docsite-v9/.storybook/main.js

```diff
@@ -3,17 +3,7 @@ const rootMain = require('../../../.storybook/main');
 
 module.exports = {
   stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(ts|tsx)', ...utils.getVnextStories()],
-  addons: [
-    {
-      name: '@storybook/addon-docs',
-      options: {
-        configureJSX: true,
-        babelOptions: {},
-        sourceLoaderOptions: null,
-        transcludeMarkdown: true,
```

Comment:
This is currently throwing an error when enabled. I couldn't see a difference between when using default options vs these custom options. Will log followup